src/main/resources/avro/biodata.avdl

```diff
@@ -0,0 +1,133 @@
+@namespace("org.ga4gh.models")
+
+/**
+This protocol defines the "biodata" objects, which can be considered data
+representations of biological correlates.
+*/
+
+protocol Biodata {
```

Comment:
The name Biodata is not very precise - everything the API currently deals with is Biodata. Biosample may be better?